LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
```

Comment:
No, this is more CRAN weirdness. Basically AFAIU, the LICENSE file cannot have the full text of standard licenses (those go in the LICENSE.md that we never send to CRAN)

https://r-pkgs.org/license.html#key-files has more details

Reply:
We don't have to have license.md at all, but it's semi-standard in the R community to set it up this way

Reply:
Ah, interesting! Then delete the LICENSE file without the .md extension?

Reply:
As strange as it sounds and looks, for the MIT license (IIRC because it has copyright holder + year) the LICENSE file is needed as well. Basically: in the license field of DESCRIPTION you can have a standard license or a file reference (for non standard licenses or extra information not allowed in the DESCRIPTION like copyright holder). And by policy the LICENSE file cannot contain the full text of a standard license. So the solution is to have both. 

Here's another example of this setup: https://github.com/r-lib/httr

We probably won't submit this to cran (at least not in the near term) so it's not a huge deal if we deviate, but one less thing to deal with if we do + it's the standard in R world for the MIT license.

Reply:
Too funny. As you were -- don't mind me ;)

PS. Thanks for taking the time to explain this all to me. Much appreciated!

Reply:
NP, I had to re-look some of this up to remind myself _why_ it's done cause it is ... strange to say the least.